dotcom-rendering/window.guardian.d.ts

```diff
@@ -68,6 +68,7 @@ declare global {
 			onConsentChange?: (fn: Callback) => void;
 			getConsentFor?: (fn: Callback) => void;
 		};
+		mockLiveUpdate: typeof onSuccess
```

Comment:
Does this need to be imported here?

Reply:
As above, I created `LiveUpdateType` instead